Validation/RecoTau/plugins/DQMHistNormalizer.cc

```diff
@@ -83,21 +68,22 @@ void DQMHistNormalizer::endRun(const edm::Run& r, const edm::EventSetup& c) {
   DQMStore& dqmStore = (*edm::Service<DQMStore>());
 
   vector<MonitorElement*> allOurMEs = dqmStore.getAllContents("RecoTauV/");
-  lat::Regexp* refregex = buildRegex("*RecoTauV/*/" + reference_);
-  vector<lat::Regexp*> toNormRegex;
+  std::regex refregex = std::regex(".*RecoTauV/.*/" + reference_, std::regex::nosubs);
+  vector<std::regex> toNormRegex;
   for (std::vector<string>::const_iterator toNorm = plotNamesToNormalize_.begin();
        toNorm != plotNamesToNormalize_.end();
        ++toNorm)
-    toNormRegex.push_back(buildRegex("*RecoTauV/*/" + *toNorm));
+    toNormRegex.emplace_back(".*RecoTauV/.*/" + *toNorm, std::regex::nosubs);
```

Comment:
@iarspider , I think we need to replace all `*` to `.*` for `toNorm` which is read from https://github.com/cms-sw/cmssw/blob/master/Validation/RecoTau/python/RecoTauValidation_cfi.py#L232